### PR TITLE
Chore: anonymous classes replaced with lambda expressions

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Part 2.

### CODE REVIEW

File: src/main/java/org/isf/patient/gui/PatientInsertExtended.java [hunk 1]
```diff
@@ -421,119 +422,44 @@ private JButton getJOkButton() {
...
+										break;
+									case JOptionPane.NO_OPTION:
+										ok = false;
+										break;
 							}
 						}
-					} else {// Update
-
+					} catch(OHServiceException ex) {
+OHServiceExceptionUtil.showMessages(ex);
```
Comment (mwithi) [inline src/main/java/org/isf/patient/gui/PatientInsertExtended.java:457]: indentation?

File: src/main/java/org/isf/patient/gui/PatientInsertExtended.java [hunk 2]
```diff
@@ -591,18 +515,96 @@ public void actionPerformed(ActionEvent e) {
...
+						}
+					}
+					patient.setNote(jNoteTextArea.getText().trim());
+
+					try {
+						patient = patientManager.savePatient(patient);
+						firePatientUpdated(patient);
+						dispose();
+					} catch(final OHServiceException ex){
+OHServiceExceptionUtil.showMessages(ex);
```
Comment (mwithi) [inline src/main/java/org/isf/patient/gui/PatientInsertExtended.java:605]: also here

File: src/main/java/org/isf/video/gui/PhotoboothComponentImpl.java [hunk 3]
```diff
@@ -241,36 +235,27 @@ public void actionPerformed(final ActionEvent actionEvent) {
...
         getOkButton().setAction(okAction);
 
-        getCancelButton().addActionListener(new ActionListener() {
-            @Override
-            public void actionPerformed(ActionEvent actionEvent) {
-                PhotoboothComponentImpl.this.presentationModel().triggerFlush();
-                PhotoboothComponentImpl.this.cleanup();
-                owner.dispose();
-            }
+        getCancelButton().addActionListener(actionEvent -> {
```
Comment (mwithi) [inline src/main/java/org/isf/video/gui/PhotoboothComponentImpl.java:238]: also `actionEvent` is a good candidate as parameter for all these cases (just a note)

**Review by mwithi — APPROVED** [review]
LGTM. Thanks!

File: src/main/java/org/isf/patient/gui/PatientInsertExtended.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/patient/gui/PatientInsertExtended.java:605]: fixed (and other stuff too)

File: src/main/java/org/isf/patient/gui/PatientInsertExtended.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/patient/gui/PatientInsertExtended.java:457]: fixed (and other stuff too)

File: src/main/java/org/isf/video/gui/PhotoboothComponentImpl.java (hunk 3 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/video/gui/PhotoboothComponentImpl.java:238]: OK; I'll see what I can do

**Pull request merged**